### ISSUE TITLE
[Feature Request] Support diffuser model: Dream

### ISSUE THREAD

**Issue by Katehuuh** [oobabooga/text-generation-webui#7147]
When run `Dream-v0-Instruct-7B-Q4_K_M.gguf` or DiffuCoder,DreamOn,Dream-Coder with `llama_cpp_binaries-0.31.0` i get: `JSON decode error: Expecting value: line 1 column 1 (char 0) Problematic line: error: {"code":500,"message":"the current context does not logits computation. skipping","type":"server_error"}`

GGUF already supports diffusion: [Dream 7B#14644](https://github.com/ggml-org/llama.cpp/pull/14644).
